suprb/optimizer/rule/mutation.py

```diff
@@ -47,6 +47,18 @@ def __call__(self, rule: Rule, random_state: RandomState) -> Rule:
         self.mutation.sigma = self.sigma
         return self.mutation(rule, random_state)
 
+    def unordered_bound(self, rule: Rule, random_state: RandomState):
+        pass
```

Comment:
If I understood `SigmaRange` correctly, we set the `RuleMutation` in init as a parameter and use this. But since we inherit from `RuleMutation` as well, which inherits from `GenerationOperator`, which has these 4 abstract methods, we need to provide some implementation, otherwise it will fail to run. But since we are not using these functions, but the ones that are specified in `__init__(self, mutation: RuleMutation = None,` we can leave them empty

Reply:
Alright, please add a short comment detailing that so we remember in a year